### ISSUE TITLE
Pairing tool team

### ISSUE THREAD

**Issue by jmbriano** [codebar/barcelona#37]
If you are interested in participating in the **Pairing tool project** describe 
[here](https://github.com/codebar/barcelona/pull/36), comment in this issue and let us know which role you would like to take :)

**Comment by jmbriano** [comment]
Greg is interested in participating.

**Comment by GMELIOT** [comment]
Hi, I'm interested in participating as a Project Leader.

**Comment by cotoflux** [comment]
I am interest in participating. Pending to see which role I would be more helpful on!

**Comment by thierrydev** [comment]
Hello
I'm very interested in participating in this project.
I'm interested in the following  areas  :
- Developers -> front & back.
- DevOps / Sysadmin.

**Comment by goyotw** [comment]
> Greg is interested in participating. Pending to talk about the role Greg wants to take

Happy to act as a PO/PM and/or a fullstack developer 🙂

**Comment by colmanfranco** [comment]
I am very interested to be envolved in the project. I can help with front or back-end technologies but also open to learn new technologies

**Comment by jmbriano** [comment]
Hi I'm Martin, It would make sense to me to play the **client** role. But if there is need i would like to be developer too to learn newer Frameworks

**Comment by Klarala13** [comment]
I am interested in participating as a developer

**Comment by jmbriano** [comment]
Hi Everyone!

We are very **VERY** happy with the interest this initiative has triggered. Also, we have more or less all the roles covered. So let's keep the inertia and kick the project off right away.

We were thinking of doing the _kickoff meeting_ next Thursday 14th after the workshop at ThoughtWorks. 

We don't want this first meeting to take too long since it could be a bit late for some team members. So if you allow me for this first time (and only for this first time), I propose the following agenda:

- Team presentations round. Individual backgrounds and desired role(s) in this project (10'), 
- Confirmation of the roles and mentors. Are we missing any role? (5'),
- Further elaboration of the requirements from [this article](https://github.com/codebar/barcelona/blob/master/tools/pairing-support-tool/README.md). Definition of the Minimum Value Product (MVP) (10'),
- Definition of communication channels (Mail, Slack, Git, other?) (5'),
- Definition of tools needed (Git, Confluence, Trello, other?) (5'),
- Next steps (Regular meetings? online work? work in smaller groups?) (5')

Summing up, it seems we will need 45 minutes for this kickoff. 

Can each one 👍 or 👎 to let me know that you saw this message, and confirming if you can participate (👍) or not (👎) on **Thursday 14th from 20:45 to 21:30** ?

Thanks a lot!
Martin

cc: @GMELIOT @cotoflux @thierrydev @goyotw @colmanfranco @Klarala13

**Comment by cotoflux** [comment]
Martin count on me 👍

**Comment by Klarala13** [comment]
👍

**Comment by ManuelagDuque** [comment]
Hello everyone... i am a little late but i would like to participate in this project. i am starting in the develop world and this community is very useful for my growth. My level of english is very basic, but i am practicing for improve. I will be to attentive to your decision. have a nice day

**Comment by hassnian** [comment]
Hi, I'm very interested in helping to this project. I can help with front and back mainly javascript and also php.

**Comment by loriking** [comment]
I'm in. Would like to work on the testing pipeline and general support with the backend stuff.

**Comment by thierrydev** [comment]
Hello @loriking @jmbriano 
Hope you are well, I would like to know when will the communication channels be setup if possible ,please ?
